rpm_spec/qubes-core-admin-client.spec.in

```diff
@@ -1,6 +1,6 @@
 Name:		qubes-core-admin-client
 Version:    @VERSION@
-Release:	1%{?dist}
+Release:	15%{?dist}
```

Comment:
This looks like your local testing.
BTW Take a look https://github.com/QubesOS/qubes-builder/blob/master/doc/Configuration.md#increment_devel_versions which does it automatically on every build.